keyboards/kinesis/kint36/keymaps/kevkevco/features/caps_word.c

```diff
@@ -0,0 +1,188 @@
+// Copyright 2022 Kevin Goldberg @kevkevco <hello@kevkev.co>
```

Comment:
If this is based on get-reuer's caps word, it should be noted that this is actually merged into core code, now.  It's on develop right now, but should hit master at the end of month.